commons/src/main/java/org/calrissian/accumulorecipes/commons/support/qfd/QfdHelper.java

```diff
@@ -141,17 +138,10 @@ public QfdHelper(Connector connector, String indexTable, String shardTable, Stor
         }
 
         if (!connector.tableOperations().exists(this.shardTable)) {
-            connector.tableOperations().create(this.shardTable, false, LOGICAL);
+            connector.tableOperations().create(this.shardTable, true);
```

Comment:
I missed this before, but did you mean to turn off Logical timestamps, and add versioning back in?